Cargo.toml

```diff
@@ -14,7 +14,7 @@ include = ["src/**/*.rs", "README.md", "LICENSE.Apache-2.0", "LICENSE.MIT"]
 
 [dependencies]
 anyhow = "1.0"
-clap = "=3.0.0-beta.5"
+clap = { version = "3.0", features = ["derive"] }
```

Comment:
oh it finally released? :grin:

Reply:
Yup :+1: , at the end of the last year i think